pawn.json

```diff
@@ -0,0 +1,23 @@
+{
+    "user": "maddinat0r",
+    "repo": "samp-discord-connector",
+    "resources": [
+        {
+            "name": "^discord-connector-(.*)-Linux.tar.gz$",
+            "platform": "linux",
+            "archive": true,
+            "includes": ["pawno/include"],
```

Comment:
The paths for the include and plugin binary is incorrect. The packaged release archive contains a root folder where those files are in. You're also missing the `log-core` library.
Check the MySQL plugin pawn.json for further reference: https://github.com/pBlueG/SA-MP-MySQL/blob/master/pawn.json

Reply:
I've updated everything, do you think this will work?